src/components/application_manager/test/commands/hmi/ui_get_capabilities_response_test.cc

```diff
@@ -0,0 +1,218 @@
+      smart_objects::SmartObject(smart_objects::SmartType_Array);
+  (*command_msg)[strings::msg_params][hmi_response::soft_button_capabilities]
+                [hmi_response::image_supported] = true;
+
+  ResponseFromHMIPtr command(
+      CreateCommand<UIGetCapabilitiesResponse>(command_msg));
+
+  EXPECT_CALL(app_mngr_, hmi_capabilities())
+      .WillOnce(ReturnRef(mock_hmi_capabilities_));
+
+  EXPECT_CALL(mock_hmi_capabilities_,
+              set_soft_button_capabilities(
+                  (*command_msg)[strings::msg_params]
+                                [hmi_response::soft_button_capabilities]));
+
+  command->Run();
+}
+
+TEST_F(UIGetCapabilitiesResponseTest, SetHmiZone_SUCCESS) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::msg_params][hmi_response::hmi_zone_capabilities] =
+      smart_objects::SmartObject(smart_objects::SmartType_Array);
+  (*command_msg)[strings::msg_params][hmi_response::hmi_zone_capabilities][0] =
+      "FRONT";
+
+  ResponseFromHMIPtr command(
+      CreateCommand<UIGetCapabilitiesResponse>(command_msg));
+
+  EXPECT_CALL(app_mngr_, hmi_capabilities())
+      .WillOnce(ReturnRef(mock_hmi_capabilities_));
+
+  EXPECT_CALL(mock_hmi_capabilities_,
+              set_hmi_zone_capabilities(
+                  (*command_msg)[strings::msg_params]
+                                [hmi_response::hmi_zone_capabilities]));
+
+  command->Run();
+}
+
+TEST_F(UIGetCapabilitiesResponseTest, SetAudioPassThru_SUCCESS) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::msg_params][strings::audio_pass_thru_capabilities] =
+      smart_objects::SmartObject(smart_objects::SmartType_Array);
+
+  ResponseFromHMIPtr command(
+      CreateCommand<UIGetCapabilitiesResponse>(command_msg));
+
+  EXPECT_CALL(app_mngr_, hmi_capabilities())
+      .WillOnce(ReturnRef(mock_hmi_capabilities_));
+
+  EXPECT_CALL(mock_hmi_capabilities_,
+              set_audio_pass_thru_capabilities(
+                  (*command_msg)[strings::msg_params]
+                                [strings::audio_pass_thru_capabilities]));
+
+  command->Run();
+}
+
+TEST_F(UIGetCapabilitiesResponseTest, SetNavigation_SUCCESS) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::msg_params][strings::hmi_capabilities] =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+  (*command_msg)[strings::msg_params][strings::hmi_capabilities]
+                [strings::navigation] = true;
+
+  ResponseFromHMIPtr command(
+      CreateCommand<UIGetCapabilitiesResponse>(command_msg));
+
+  EXPECT_CALL(app_mngr_, hmi_capabilities())
+      .WillOnce(ReturnRef(mock_hmi_capabilities_));
+
+  EXPECT_CALL(mock_hmi_capabilities_,
+              set_navigation_supported(
+                  (*command_msg)[strings::msg_params][strings::hmi_capabilities]
+                                [strings::navigation].asBool()));
```

Comment:
@wolfylambova22, better to move value of `(*command_msg)[strings::msg_params][strings::hmi_capabilities][strings::navigation]` to the separate variable with clear name and then use it.